wizard/WizardController.qml

```diff
@@ -434,7 +434,7 @@ Rectangle {
 
     function creatingWalletDeviceSplash(){
         var splashMsg = qsTr("Creating wallet from device...");
-        splashMsg += wizardController.walletOptionsDeviceName === "Ledger" ? qsTr("\n\nPlease check your hardware wallet –\nyour input may be required.") : "";
+        splashMsg += wizardController.walletOptionsDeviceName === "Ledger" ? qsTr("\n\nPlease check your hardware wallet: –\nit is strongly recommended to export the view key.") : "";
```

Comment:
```suggestion
        splashMsg += wizardController.walletOptionsDeviceName === "Ledger" ? qsTr("\n\nPlease check your hardware wallet –\nand export the view key.") : "";
```

As far as I know it's broken without exporting the view key so I would just use this. Also the `:` you added seems wrong.